scripts/base/utils/site.zeek

```diff
@@ -103,10 +103,15 @@ export {
 		[fe80::]/10,
 	};
 
-	## Networks that are considered "local".  Note that ZeekControl sets
-	## this automatically.
```

Comment:
I'd keep the ZeekControl comment: That's not only about private address space. ZeekControl users normally configure all of their local nets over there.

Reply:
Yes that's true. I removed it because of the upcoming zeekctl change that pairs with this, but you're right — it's still worth flagging the connection. Will do.